deepchem/molnet/load_function/pdbbind_datasets.py

```diff
@@ -126,3 +133,92 @@ def load_pdbbind_grid(split="random",
                                                transformers)
 
     return tasks, (train, valid, test), transformers
+
+def load_pdbbind(featurizer="grid", split="random", subset="core", reload=True):
+  """Load and featurize raw PDBBind dataset."""
+  pdbbind_tasks = ["-logKd/Ki"]
+  data_dir = deepchem.utils.get_data_dir()
+  if reload:
+    save_dir = os.path.join(data_dir, "pdbbind/" + featurizer + "/" + str(split))
+    loaded, all_dataset, transformers = deepchem.utils.save.load_dataset_from_disk(
+        save_dir)
+    if loaded:
+      return pdbbind_tasks, all_dataset, transformers
+  dataset_file = os.path.join(data_dir, "pdbbind_v2015.tar.gz")
+  data_folder = os.path.join(data_dir, "v2015")
+  if not os.path.exists(dataset_file):
+    logger.warning("About to download PDBBind full dataset. Large file, 2GB")
+    deepchem.utils.download_url(
+        'http://deepchem.io.s3-website-us-west-1.amazonaws.com/datasets/' +
+        "pdbbind_v2015.tar.gz")
+  if os.path.exists(data_folder):
+    logger.info("Data directory for %s already exists" % subset)
+  else:
+    print("Untarring full dataset")
+    deepchem.utils.untargz_file(dataset_file, dest_dir=data_dir)
+  if subset == "core":
+    index_file = os.path.join(data_folder, "INDEX_core_name.2013")
+    labels_file = os.path.join(data_folder, "INDEX_core_data.2013")
+  elif subset == "refined":
+    index_file = os.path.join(data_folder, "INDEX_refined_name.2013")
+    labels_file = os.path.join(data_folder, "INDEX_refined_data.2013")
+  else:
+    raise ValueError("Other subsets not supported")
+  # Extract locations of data
+  pdbs = []
+  with open(index_file, "r") as g:
+    lines = g.readlines()
+    for line in lines:
+      line = line.split(" ")
+      pdb = line[0]
+      if len(pdb) == 4:
+        pdbs.append(pdb)
+  protein_files = [os.path.join(data_folder, pdb, "%s_protein.pdb" % pdb)
+               for pdb in pdbs]
+  ligand_files = [os.path.join(data_folder, pdb, "%s_ligand.sdf" % pdb)
+               for pdb in pdbs]
+  # Extract labels 
+  labels = []
+  with open(labels_file, "r") as f:
+    lines = f.readlines()
+    for line in lines:
+      # Skip comment lines
+      if line[0] == "#":
+        continue
+      # Lines have format
+      # PDB code, resolution, release year, -logKd/Ki, Kd/Ki, reference, ligand name
+      line = line.split()
+      # The base-10 logarithm, -log kd/pk
+      log_label = line[3]
+      labels.append(log_label)
+  # Featurize Data
+  if featurizer == "grid":
+    # TODO: This is not the correct setting. Set hyperparameters correctly 
+    ecfp_power = 5
+    splif_power = 5
+    featurizer = rgf.RdkitGridFeaturizer(
```

Comment:
@lilleswing This invocation isn't the one we used for the pre-computed featurizations. Do you happen to remember what the settings we used were?